lib/rex/post/meterpreter/extensions/stdapi/fs/dir.rb

```diff
@@ -194,9 +195,9 @@ def Dir.unlink(path)
   # Downloads the contents of a remote directory a
   # local directory, optionally in a recursive fashion.
   #
-  def Dir.download(dst, src, recursive = false, force = true, &stat)
+  def Dir.download(dst, src, recursive = false, force = true, glob = nil, &stat)
```

Comment:
mmm it's changing a method signature, and later in the same method method I see:

```ruby
stat.call('mirroring', src_item, dst_item) if (stat)
download(dst_item, src_item, recursive, force, &stat) # It looks suspicious to me
stat.call('mirrored', src_item, dst_item) if (stat)
```

Is it a recursive call? If it is looks like the new signature will break the recursive call, won't it?

Maybe I'm forgetting something, feel free to clarify. On the other hand. Did you verify there aren't ```Dir.download``` calls which could become broken with the new signature? Just asking.